lib/components/root/entity.vue

```diff
@@ -0,0 +1,131 @@
+<!--
```

Comment:
tbh way more elaborate than i'd have personally gone for but you made it work and it seems to work good